x-pack/plugins/serverless_search/public/application/components/connectors_router.tsx

```diff
@@ -9,10 +9,15 @@ import { Route, Routes } from '@kbn/shared-ux-router';
 import React from 'react';
 import { EditConnector } from './connectors/edit_connector';
 import { ConnectorsOverview } from './connectors_overview';
+import { ConnectorsElasticManaged } from './connectors_elastic_managed';
+import { ELASTIC_MANAGED_CONNECTOR_PATH } from '../constants';
 
 export const ConnectorsRouter: React.FC = () => {
   return (
     <Routes>
+      <Route exact path={ELASTIC_MANAGED_CONNECTOR_PATH}>
```

Comment:
I'm not actually sure here, this should just work as you've written it. But are we sure that the presence or absence of a created connector is actually the issue? Because that really shouldn't be affecting this.